account_asset_analytic/models/account_asset_depreciation_line.py

```diff
@@ -9,8 +9,9 @@ class AccountAssetDepreciationLine(models.Model):
     _inherit = 'account.asset.depreciation.line'
 
     @api.multi
-    def create_move(self):
-        move_ids = super(AccountAssetDepreciationLine, self).create_move()
+    def create_move(self, post_move=True):
+        move_ids = super(AccountAssetDepreciationLine, self).create_move(
+            post_move=True)
```

Comment:
This should be `post_move=post_move` (the value received by argument). One thing is the default value (True), which is used in the signature in case this argument is not passed, and the other is the argument passed to super, which is the value contained on `post_move` (which can be the default or not, but that doesn't matter you).